File: DietitianHub-Backend/src/main/java/gr/unipi/thesis/dimstyl/dtos/api/HomeResponse.java
package gr.unipi.thesis.dimstyl.dtos.api;

import lombok.Builder;

import java.util.List;

@Builder
public record HomeResponse(String fullName,
                           List<ApiArticleDto> articles,
                           List<ApiAnnouncementDto> announcements,
                           List<ApiAppointmentDto> appointments) {
}

File: DietitianHub-Android/app/src/main/java/gr/unipi/thesis/dimstyl/data/models/HomeResponse.kt
package gr.unipi.thesis.dimstyl.data.models

import gr.unipi.thesis.dimstyl.domain.models.Announcement

data class HomeResponse(
    val fullName: String,
    val articles: List<Article>,
    val announcements: List<Announcement>,
    val appointments: List<Appointment>
)